contracts/floaty/src/contract.rs

```diff
@@ -53,16 +51,16 @@ pub fn execute(
         let mut fl = balance[0].amount.u128() as f64;
         fl *= 0.3;
 
-        let mut resp = Response::new();
-        resp.add_attribute("action", "release");
```

Comment:
I was going to say there was already such functions.

This changes the usage from `&mut self` to `self`, which I think is a good change :+1: